mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -90,7 +90,7 @@ final class OneVsRestModel private[ml] (
     val accColName = "mbc$acc" + UUID.randomUUID().toString
     val init: () => Map[Int, Double] = () => {Map()}
     val mapType = MapType(IntegerType, DoubleType, valueContainsNull = false)
-    val newDataset = dataset.withColumn(accColName, callUDF(init, mapType))
+    val newDataset = dataset.withColumn(accColName, udf(init).apply())
```

Comment:
Style nit: I think we call `udf` above so the invocation just looks like `init()`.